system/indy-node-tests/TestCatchUpSuiteExtended.py

```diff
@@ -14,8 +14,8 @@ class TestCatchUpSuiteExtended:
     @pytest.mark.parametrize(
         'wait_catchup_before_ordering, main_txn_count, extra_txn_count',
         [
-            (True, 55, 0),  # wait for catchup subcase
-            (False, 10, 0),  # less than 100 requests total subcase
+            # (True, 55, 0),  # wait for catchup subcase
+            # (False, 10, 0),  # less than 100 requests total subcase
```

Comment:
Having a short comment on why we skip those cases would be helpful

Reply:
Done and uncomment the first parameter since we wait for catchup there.